test/test_quadruplets_classifiers.py

```diff
@@ -25,35 +25,11 @@ def test_predict_only_one_or_minus_one(estimator, build_dataset,
   assert np.isin(predictions, [-1, 1]).all()
 
 
-@pytest.mark.parametrize('with_preprocessor', [True, False])
```

Comment:
I removed this test because it makes no sense for quadruplets since the threshold should be always 0